poutyne/framework/experiment.py

```diff
@@ -331,24 +518,58 @@ def load_best_checkpoint(self, *, verbose=False):
         return best_epoch_stats
 
     def load_checkpoint(self, epoch):
+        """
```

Comment:
Maybe this function should verify that the checkpoint exists and raise an Exception if it doesn't instead of getting an IO error.

Reply:
I guess I agreed.

Reply:
Agreed. Fixed by 4efa7476e936712a8a4fa3c9ff8957e924f97642.